sw/device/silicon_creator/lib/drivers/otbn.c

```diff
@@ -144,7 +144,8 @@ static rom_error_t otbn_cmd_run(otbn_cmd_t cmd, rom_error_t error) {
   uint32_t reg = 0;
   do {
     reg = abs_mmio_read32(kBase + OTBN_INTR_STATE_REG_OFFSET);
-    res ^= bitfield_bit32_read(reg, OTBN_INTR_COMMON_DONE_BIT) << kResDoneBit;
+    res ^= (unsigned int)bitfield_bit32_read(reg, OTBN_INTR_COMMON_DONE_BIT)
```

Comment:
would the following work?
```suggestion
    res ^= (uint32_t)bitfield_bit32_read(reg, OTBN_INTR_COMMON_DONE_BIT)
```

Reply:
Done.

The issue this address is `Implicit conversion changes signedness: 'int' to 'unsigned int' [-Wsign-conversion]`, so I made a mechanical change.

I felt a little uneasy about XORing this value into `rom_error_t` because enums are `int`. But I think either way this is fine because `res` will undergo implicit conversion to `unsigned int` before it gets XORed with the RHS `uint32_t`. https://en.cppreference.com/w/c/language/conversion#Usual_arithmetic_conversions